azkaban-common/src/test/bash/test_util.shunit2

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env shunit2
+# Make sure to "brew install shunit2" or use your favorite package manager to install
+# before running tests
+
+function oneTimeSetUp() {
```

Comment:
Did you put the source inside a function to prevent accidental execution of the code from the external file?